docs/src/modules/ROOT/pages/constraints-and-score/score-calculation.adoc

```diff
@@ -1498,21 +2469,25 @@ public class MyConstraintProviderTest {
 Timefold Solver supports two other types of score calculation.
 
 [#easyJavaScoreCalculation]
-=== Easy Java score calculation
+=== Easy Java/Python score calculation
```

Comment:
Same comment as above; specifically naming the technology seems unnecessary. The heading, as well as the references, should change.